pyproject.toml

```diff
@@ -101,7 +103,7 @@ all = ["skema[core]", "skema[dev]", "skema[doc]", "skema[demo]", "skema[annotati
 
 [tool.setuptools.package-data]
 # needed to ensure models are included in package/discoverable
-"*" = ["*.json", "vocab.txt", "*.pt", "*.png"]
+"*" = ["*.json", "vocab.txt", "*.pt", "*.png", "*.html", "*.yml", "*.yaml"]
```

Comment:
@vincentraymond-ua , do we need to include .html in the package source? Site publishing ignores these settings.

Reply:
nm.  I see why this is needed now.